src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -166,9 +170,9 @@ public void parseTag_validValueWithoutWhitespace_returnsTag() throws Exception {
 
     @Test
     public void parseTag_validValueWithWhitespace_returnsTrimmedTag() throws Exception {
-        String tagWithWhitespace = WHITESPACE + VALID_TAG_1 + WHITESPACE;
+        String appointmentWithWhitespace = WHITESPACE + VALID_TAG_1 + WHITESPACE;
```

Comment:
Did you change the test
public void parseTag_validValueWithWhitespace_returnsTrimmedTag()
to a one for appointmentWithWhitespace?

Reply:
Where did the method go?

Reply:
Why is parseTag called on appointmentWithWhitespace?